frontend/src/pages/member/LoginPage.vue

```diff
@@ -73,12 +73,31 @@ export default {
         return;
       }
       try {
-        // TODO login API를 작성해주세요.
-        // const { email, password } = this.member;
-        // const data = await fetch("/login")
-        // TODO member 데이터를 불러와 주세요.
-        // const member = wait fetch("/members/me")
-        // this.setMember(member);
+        // TODO login API를 작성해주세요. (v)
+        const { email, password } = this.member;
+        const data = await fetch("http://localhost:8080/login/token", {
+          method: "POST",
+          headers: {
+            "Content-Type" : "application/json"
+          },
+          body: JSON.stringify({
+            email: email,
+            password: password
+          })
+        })
+        const token = await data.json();
+        localStorage.setItem("token", token["accessToken"]);
+
+        // TODO member 데이터를 불러와 주세요. (v)
+        const response  = await fetch("http://localhost:8080/members/me", {
+          headers: {
+            "Authorization": "Bearer " + token["accessToken"]
```

Comment:
`token["accessToken"]` 을 사용해도 되지만 `localStorage.getItem("token")`를 써도 될 것 같아요. 
다른 곳에서 토큰을 꺼내는 작업과 통일성을 위해 `localStorage.getItem("token")` 사용을 고려해봐도 좋을 것 같아요 :) 
```suggestion
            "Authorization": "Bearer " + localStorage.getItem("token")
```

Reply:
통일시켜주었습니다! :